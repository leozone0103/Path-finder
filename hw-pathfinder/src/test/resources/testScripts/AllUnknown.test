# Test when both nodes are unknown.
CreateGraph graph
AddNode graph A
FindPath graph C D
FindPath graph M N