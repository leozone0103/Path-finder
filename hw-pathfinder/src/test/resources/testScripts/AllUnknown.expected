# Test when both nodes are unknown.
created graph graph
added node A to graph
unknown: C
unknown: D
unknown: M
unknown: N